### PR TITLE
feat(51 link loginregister mobile > main): add navigation between pages and edit and add component

### PR DESCRIPTION
**Pull request by Kiya971**
- Navigation between Login Page and Area Page
- Add Component
- Edit Component

### CODE REVIEW

File: mobile/src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+              placeholder='Password'
+              placeholderTextColor={'rgba(43, 34, 119, 1)'}
+              onChangeText={newPassword => setPassword(newPassword)}
+              defaultValue={password}
+              secureTextEntry={true}
+            />
+          </View>
+          <View style={{paddingHorizontal: 50}}>
+            <Pressable style={styles.primaryButton} onPress={() => { 
+              console.log('pressed')
```
Comment (AlexGuillard) [inline mobile/src/pages/Login.tsx:52]: delete debug

**Review by AlexGuillard — CHANGES_REQUESTED** [review]
delete debug

**Review by AlexGuillard — DISMISSED** [review]
ok

**Review by Miou-zora — APPROVED** [review]

**Review by Benjicatch — APPROVED** [review]
ok

**Review by Pablodeibar — APPROVED** [review]
ok

**Pull request merged**